cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,80 @@
+from collections import defaultdict
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_list_arg
+
+
+class SetFieldHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "fields": {
+            "description": "List of object fields to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
+        super()._init_options(kwargs)
+
+        try:
+            float(self.api_version)
+        except ValueError:
+            raise TaskOptionsError(f"Invalid API version {self.api_version}")
+
+        if "fields" not in self.options:
+            raise TaskOptionsError(
+                "The 'fields' option is required, please pass a dictionary with the api_name and help_text keys."
+            )
```

Comment:
BaseTask takes care of validating that options with `"required": True` are provided, so I don't think this check is needed.